platforms/ios/build_framework.py

```diff
@@ -32,34 +32,16 @@
 """
 
 from __future__ import print_function, unicode_literals
-import glob, re, os, os.path, shutil, string, sys, argparse, traceback, multiprocessing
+from builtins import str
+from builtins import object
```

Comment:
These should be removed - the `builtins` module requires the `future` package to be installed on Python 2.7, which we don't want.

Reply:
👍